roles/pulp_workers/tasks/main.yml

```diff
@@ -12,6 +12,26 @@
 - name: Get services
   service_facts:
 
+- name: pulpcore resource manager
+  block:
+
+    - name: Disable pulpcore-resource-manager service
+      systemd:
+        name: pulpcore-resource-manager.service
+        state: stopped
+        enabled: false
+        daemon_reload: true
+
+    - name: Remove pulpcore-resource-manager service
+      file:
+        path: /lib/systemd/system/pulpcore-resource-manager.service
+        state: absent
+
+  become: true
+  when:
+    - ansible_facts.services['pulpcore-resource-manager.service'] is defined
+    - __pulp_version is version('3.13', '>=')
```

Comment:
I don't know the best way to handle upgrade scenarios,
if I do:
```
__pulp_version is version('3.16', '>=')
```
the release-upgrade and package-upgrade break!
I'm considering 3.13 as it is the first release with the new tasking system

FYI: @mdellweg @mikedep333